src/closure-types.ts

```diff
@@ -36,6 +35,25 @@ export function closureTypeToTypeScript(closureType: string): string {
 }
 
 /**
+ * Convert from a parameter type annotation in Closure syntax to TypeScript
+ * syntax (e.g `Array=` => `{type: 'Array<any>|null', optional: true}`).
+ */
+export function closureParamToTypeScript(closureType: string):
+    {optional: boolean, type: string} {
+  let ast;
+  try {
+    ast = parseParamType(closureType);
+  } catch {
+    return {optional: true, type: 'any'};
```

Comment:
Optional as the fallback here will cause trouble if early params are unparsable, because non-optional arguments can't follow optional ones.

I think non-optional but any-typed is the better fallback. Or do `optional: closureType.endsWith('=')` to get 99% of cases correct

Reply:
Good call, done (with the `endsWith`).